omod/src/test/java/org/openmrs/module/Oauth2ExceptionResolverTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) Amiyul LLC - All Rights Reserved
+ *
+ * This source code is protected under international copyright law. All rights
+ * reserved and protected by the copyright holder.
+ *
+ * This file is confidential and only available to authorized individuals with the
+ * permission of the copyright holder. If you encounter this file and do not have
+ * permission, please contact the copyright holder and delete this file.
+ */
+package org.openmrs.module;
```

Comment:
Since the class you are testing is in the `org.openmrs.module.oauth2login.web` package. Shouldn't the test be in the same?

Reply:
Good catch! I have moved it